### PR TITLE
Assigment2

### PR DESCRIPTION
**Pull request by pgandoy**
PabloGandoy

### CODE REVIEW

File: Assignment2/PabloGandoy.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+],
+  "subjects":[
+    {
+      "name":"Semantic Web and Linked Data",
+      "description":"Very interesting"
+      "teachers":["Oscar Corcho","Paola Espinoza"]
+    }
+  ],
+  ]
+}
```
Comment (paoespinozarias) [inline Assignment2/PabloGandoy.json:25]: This file has syntax erros. Validate it before resubmiting.

File: Assignment2/PabloGandoyJSONtoJSONLD.json-ld [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+},
+  "subjects":[
+    {
+      "name":"Semantic Web and Linked Data",
+      "description":"Very interesting"
+      "teachers":["Oscar Corcho","Paola Espinoza"]
+    }
+  ],
+  ]
+}
```
Comment (paoespinozarias) [inline Assignment2/PabloGandoyJSONtoJSONLD.json-ld:30]: This file has syntax erros. Validate it before resubmiting.

**Review by paoespinozarias — CHANGES_REQUESTED** [review]

**Comment by ocorcho** [comment]
Assignment 2a is ok

File: Assignment2/PabloGandoyRDFtoJSONLD.json-ld [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      "http://example.org/properties/hasMeasurement": {
+        "@id": "http://example.org/properties/Measurement8401"
+      }
+      {
+        "@id": "http://example.org/properties/User10A",
+        "http://example.org/properties/hasName": "Pedro"
+      }
+    }
+  ]
+}
```
Comment (paoespinozarias) [inline Assignment2/PabloGandoyRDFtoJSONLD.json-ld:45]: This file has syntax errors. Validate it before resubmitting.

**Review by paoespinozarias — CHANGES_REQUESTED** [review]

File: Assignment2/PabloGandoyJSONtoJSONLD.json-ld [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{
+@context:{
```
Comment (paoespinozarias) [inline Assignment2/PabloGandoyJSONtoJSONLD.json-ld:2]: Missing " "

File: Assignment2/PabloGandoyJSONtoJSONLD.json-ld [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  "tipo":"rdfs:label",
+  "estado":"rdfs:comment",
+  "name":"vocab:name",
+  "hometown":"vocab:hometown",
+  "age":"vocab:age",
+  "almamater":"vocab:almamater",
+  "parents":"vocab:parents",
+  "subjects":"vocab:subjects"
+},
+"name":"Pablo Gandoy",
```
Comment (paoespinozarias) [inline Assignment2/PabloGandoyJSONtoJSONLD.json-ld:16]: Include a type and id

**Review by paoespinozarias — CHANGES_REQUESTED** [review]

File: Assignment2/PabloGandoyRDFtoJSONLD.json-ld (hunk 3 above)
Comment (paoespinozarias) [reply Assignment2/PabloGandoyRDFtoJSONLD.json-ld:45]: Now, it is ok!

**Comment by ocorcho** [comment]
assignment 2a is now fixed.

**Comment by ocorcho** [comment]
Requests done by @paoespinozarias have not been fixed, so we close this pull request.

**Pull request closed without merging**